src/scripts/webpack/config.js

```diff
@@ -48,19 +48,22 @@ const getConfigType = (configFolderPath) => {
 };
 
 const nodeEnv = process.env.NODE_ENV || 'development';
-const envWhitelist = ['apiEndpoint', 'fetchLocalesFrom'];
```

Comment:
Uh also, noticed after the previous comment: `"debug": "scriptoni:*"` is not in my config as well